app/models/consistency_checklist.rb

```diff
@@ -73,4 +75,8 @@ class ConsistencyChecklist < ApplicationRecord
       )
     end
   end
+
+  def reset_site_map_correct_comment_if_needed
+    self.site_map_correct_comment = nil if site_map_correct == "yes"
```

Comment:
Should `site_map_correct` really be storing a string? Can't we have that as a boolean?

I realise this isn't being introduced by this PR and it's an existing issue, but it doesn't seem ideal.

Reply:
Yes, unfortunately is not boolean and for this reason I did this.